src/wp-includes/compat.php

```diff
@@ -374,7 +374,44 @@ function is_iterable( $var ) {
 		return ( is_array( $var ) || $var instanceof Traversable );
 	}
 }
+if ( ! function_exists( 'array_key_first' ) ) {
+	/**
+	 * Polyfill for array_key_first() function added in PHP 7.3.
+	 *
+	 * Get the first key of the given array without affecting the internal array pointer.
+	 *
+	 * @ticket 45055
+	 * @ignore
```

Comment:
- The `@ticket` annotation is not needed.
- The `@ignore` annotation is not needed.